src/fortran/EPOS-LHC/sources/epos-bas-lhc.f

```diff
@@ -2831,7 +2831,7 @@ subroutine ainit
       inicnt=inicnt+1
 
       if(inicnt.eq.1)then
-        write(ifmt,'(a)')'initializations ...'
+c        write(ifmt,'(a)')'initializations ...'
```

Comment:
I think it is fine to make trivial changes like this, since this is not compromising the results of EPOS-LHC in any way. I think Anatoli also made changes to models by adding impy logging functions.